go.mod

```diff
@@ -121,6 +121,7 @@ require (
 	k8s.io/apimachinery v0.24.3
 	k8s.io/client-go v0.24.3
 	k8s.io/component-base v0.24.3
+	k8s.io/kubernetes v1.13.0
```

Comment:
there's a non-canonical api import somewhere here that is adding this as a dependency

Reply:
Yeah it's a hangover from when I imported storage incorrectly.  I'll fix it up.